app/models/response.rb

```diff
@@ -1,59 +1,103 @@
 # frozen_string_literal: true
-
 class Response < ApplicationRecord
   include ScorableHelper
   include MetricHelper
+  include ResponseHelper
 
   belongs_to :response_map, class_name: 'ResponseMap', foreign_key: 'map_id', inverse_of: false
-  has_many :scores, class_name: 'Answer', foreign_key: 'response_id', dependent: :destroy, inverse_of: false
-
-  alias map response_map
-  delegate :questionnaire, :reviewee, :reviewer, to: :map
-
-  def reportable_difference?
-    map_class = map.class
-    # gets all responses made by a reviewee
-    existing_responses = map_class.assessments_for(map.reviewee)
-
-    count = 0
-    total = 0
-    # gets the sum total percentage scores of all responses that are not this response
-    existing_responses.each do |response|
-      unless id == response.id # the current_response is also in existing_responses array
-        count += 1
-        total +=  response.aggregate_questionnaire_score.to_f / response.maximum_score
-      end
+  has_many :scores, class_name: 'Answer'
```

Comment:
Please add the dependent destroy, foriegn_key and inverse_of attributes as it was present previously.